cid/common.py

```diff
@@ -1279,8 +1279,15 @@ def map(self, **kwargs):
             self.accountMap.create(v)
 
     @command
-    def init(self, **kwargs):
-        """ Initialize account resources for deployment """
-        from commands import InitCommand
-        cmd = InitCommand(cid=self, **kwargs)
+    def initqs(self, **kwargs):
+        """ Initialize QuickSight resources for deployment """
+        from commands import InitQsCommand
```

Comment:
is there a reason to bring this import to function level. Can lead to errros that are hard to debug

Reply:
also probably you mean

from cid.commands import InitQsCommand

Reply:
I'm using type hints and leaving the InitQsCommand as a top level import will lead to circular dependencies